sklearn/utils/estimator_checks.py

```diff
@@ -710,7 +710,7 @@ def check_classifiers_one_label(name, Classifier):
 
 
 def check_classifiers_train(name, Classifier):
-    X_m, y_m = make_blobs(random_state=0)
```

Comment:
why is this needed? I would rather not make test run slower unless we have a really good reason.